L66-core-cancellation-status.md

```diff
@@ -0,0 +1,60 @@
+Core server RPCs will not report cancellation if completed with non-OK status
+----
+* Author(s): vpai
+* Approver: markdroth
+* Status: Proposed
+* Implemented in: https://github.com/grpc/grpc/pull/22991
+* Last updated: May 28, 2020
+* Discussion at https://groups.google.com/g/grpc-io/c/5o3EDPqz9is
+
+## Abstract
+
+Clarify that gRPC core will not mark server RPCs cancelled if they explicitly close with `GRPC_OP_SEND_STATUS_FROM_SERVER` using a non-`OK` status.  This is technically a core API change and thus needs a major revision increment.
+
+## Background
+
+gRPC core versions up to 10.X specify that the out argument of the `GRPC_OP_RECV_CLOSE_ON_SERVER` operation is a pointer to an int that specifies whether the RPC was cancelled. The definition specified in the comment is
+
+```
+      /** out argument, set to 1 if the call failed in any way (seen as a
+          cancellation on the server), or 0 if the call succeeded */
+```
+
+The term `in any way` is not clear, and the broadest (and current) definition is that it accounts for failures caused by client-side or server-side cancellations, deadline exceeded, maximum connection age exceeded, network resets, *and also explicit sending of non-OK status*.
```

Comment:
Minor:

```suggestion
The term `in any way` is not clear, and the broadest (and current) definition is that it accounts for failures caused by client-side or server-side cancellations, deadlines being exceeded, maximum connection age being exceeded, network resets, *and also explicit sending of non-OK status*.
```

I couldn't think of a better way to "noun" "exceeded"... "Expirations"? But that doesn't really work for max connection age...

Reply:
connections exceeding their maximum age ?

Reply:
SGTM